apps/css-workshop/src/pages/checkbox.astro

```diff
@@ -78,435 +55,228 @@ import Layout from './_layout.astro';
-      </div>
-      <div class='iui-status-message' data-iui-status='negative'>
-        <div class='iui-svg-icon'>
-          <svg-status-error class='iui-svg-icon' aria-hidden='true'></svg-status-error>
+      <div class='iui-input-grid iui-input-group-wrapper'>
+        <div class='iui-input-label'>Checkbox group label</div>
+        <div class='iui-input-group'>
+          <Checkbox_ label='Option 1' checked />
+          <Checkbox_ label='Option 2' />
+          <Checkbox_ label='Option 3' disabled checked />
+          <Checkbox_ label='Option 4' disabled />
+        </div>
+        <div class='iui-status-message' data-iui-status='negative'>
+          <div class='iui-svg-icon'>
+            <svg-status-error class='iui-svg-icon' aria-hidden='true'></svg-status-error>
+          </div>
+          <div>Negative message.</div>
         </div>
-        <div>Negative message.</div>
       </div>
-    </div>
 
-    <div class='iui-input-grid iui-input-group-wrapper'>
-      <div class='iui-input-label'>Checkbox group label</div>
-      <div class='iui-input-group'>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' checked />
-          <span class='iui-checkbox-label'>
-            <svg-smiley-happy-very aria-hidden='true' style='display: flex;'
-            ></svg-smiley-happy-very>
-          </span>
-        </label>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' />
-          <span class='iui-checkbox-label'>
+      <div class='iui-input-grid iui-input-group-wrapper'>
+        <div class='iui-input-label'>Checkbox group label</div>
+        <div class='iui-input-group'>
+          <Checkbox_ checked>
             <svg-smiley-happy aria-hidden='true' style='display: flex;'></svg-smiley-happy>
-          </span>
-        </label>
-        <label class='iui-checkbox-wrapper iui-disabled'>
-          <input class='iui-checkbox' type='checkbox' checked disabled />
-          <span class='iui-checkbox-label'>
+          </Checkbox_>
+          <Checkbox_>
+            <svg-smiley-happy aria-hidden='true' style='display: flex;'></svg-smiley-happy>
+          </Checkbox_>
+          <Checkbox_ checked disabled>
             <svg-smiley-sad aria-hidden='true' style='display: flex;'></svg-smiley-sad>
-          </span>
-        </label>
-        <label class='iui-checkbox-wrapper iui-disabled'>
-          <input class='iui-checkbox' type='checkbox' disabled />
-          <span class='iui-checkbox-label'>
+          </Checkbox_>
+          <Checkbox_ disabled>
             <svg-smiley-sad-very aria-hidden='true' style='display: flex;'></svg-smiley-sad-very>
-          </span>
-        </label>
-      </div>
-      <div class='iui-status-message'>
-        <div>Help message.</div>
+          </Checkbox_>
+        </div>
+        <div class='iui-status-message'>
+          <div>Help message.</div>
+        </div>
       </div>
     </div>
```

Comment:
I believe this should still be `</section>`. i.e. the `demo-default` section should end here, like it was previously.

A similar change would be needed for L281.

Reply:
addressed this in my last push